src/adult_mosquito_ode.h

```diff
@@ -24,7 +24,7 @@ struct AdultMosquitoModel {
     double mu; //death rate for adult female mosquitoes
     const double tau; //extrinsic incubation period
     double foim; //force of infection towards mosquitoes
-    AdultMosquitoModel(MosquitoModel, double, double, double);
+    AdultMosquitoModel(MosquitoModel, double, double, double, double);
```

Comment:
should this be `int` or `size_t` here for max steps?

Reply:
I see you use `size_t` below in `src/mosquito_ode.cpp`

Reply:
Ah, it's a bit confusing because there are several classes at play here. I've made an issue to improve this documentation #132 

The `size_t` below is for the `Solver`. Whereas the `double` here is for the `AdultMosquitoModel`.

We've decoupled the model from the solver, to help switch between individual mosquitoes and compartmental ones.